cmake/partition_manager.cmake

```diff
@@ -449,36 +503,35 @@ else()
           ${${name}}
           )
       endforeach()
-
-      if (CONFIG_NRF53_UPGRADE_NETWORK_CORE
-          AND CONFIG_HCI_RPMSG_BUILD_STRATEGY_FROM_SOURCE)
-          # Create symbols for the offset reqired for moving the signed network
-          # core application to MCUBoots secondary slot. This is needed
-          # because  objcopy does not support arithmetic expressions as argument
-          # (e.g. '0x100+0x200'), and all of the symbols used to generate the
-          # offset are only available as a generator expression when MCUBoots
-          # cmake code exectues.
-
-          get_target_property(
-            net_app_addr
-            partition_manager
-            CPUNET_PM_APP_ADDRESS
-            )
-
-          # There is no padding in front of the network core application.
-          math(EXPR net_app_TO_SECONDARY
-            "${PM_MCUBOOT_SECONDARY_ADDRESS} - ${net_app_addr} + ${PM_MCUBOOT_PAD_SIZE}")
-
-          set_property(
-            TARGET partition_manager
-            PROPERTY net_app_TO_SECONDARY
-            ${net_app_TO_SECONDARY}
-            )
-        endif()
-
     endif()
   endforeach()
 
+  if (CONFIG_BOOTLOADER_MCUBOOT)
+    if (CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY)
+      get_filename_component(qspi_node ${ext_flash_dev} DIRECTORY)
```

Comment:
if user has not migrated to the devicetree external flash feature, then this will fail, as far as i can tell, cause in that case `ext_flash_dev` has not been defined.

Reply:
I will add a check for its definition.

Reply:
Actually no, this config can only be set if you have used the devicetree overlay.

![image](https://user-images.githubusercontent.com/1547348/131664641-8739da62-306e-471e-8ed9-0528b9a24cb9.png)

Reply:
If MCUBOOT is enabled, then `CONFIG_BOOTLOADER_MCUBOOT` cannot be enabled.

Reply:
![image](https://user-images.githubusercontent.com/1547348/131964435-a68131a6-7ac6-4640-bf7b-1afb546eeb20.png)